coral/src/app/features/approvals/schemas/components/SchemaApprovalsTable.test.tsx

```diff
@@ -45,10 +44,9 @@ const mockedRequests: SchemaRequest[] = [
     environmentName: "SEC",
     schemaversion: "1.0",
     teamname: "NCC1701D",
-    teamId: 1701,
+    teamId: 1702,
```

Comment:
captain of the NCC-1702?? 😱

Reply:
(I know why that was necessary 😅  btw)

Reply:
Haha yes sorry! I think it was necessary but maybe not!